<launch>

<!-- Detection threshold. Default value = 0.0 -->
<param name="people/pedestrian_detector_HOG/hit_threshold" type="double" value="0.0"/>
<!-- Minimum number of close-together detections required for each returned person. Default=2 -->
<param name="people/pedestrian_detector_HOG/group_threshold" type="int" value="2"/>
<!-- Use any of the depth-related functions -->
<param name="people/pedestrian_detector_HOG/use_depth" type="bool" value="false"/>
<!-- Use height to crop the image -->
<param name="people/pedestrian_detector_HOG/use_height" type="bool" value="false"/>
<!-- If height is used, this is the maximum height allowed, in meters -->
<param name="people/pedestrian_detector_HOG/max_height_m" type="double" value="2.2"/>
<!-- Frame wrt which the height is calculated. -->
<param name="people/pedestrian_detector_HOG/ground_frame" type="string" value="base_link" />
<!-- Display the detections -->
<param name="people/pedestrian_detector_HOG/do_display" type="bool" value="false"/>

<node pkg="pedestrian_detector_HOG" type="pedestrian_detector_HOG" output="screen"/>

</launch>
